cpplib/Tests/GeometryTest.cpp

```
@@ -831,164 +829,230 @@ TEST_F(VoronoiTest, AlexTest) {
-                                                                  {0.5699,  0.0585,  0.1577 },
-                                                                  {0.6972,  0.22032, 0.31696},
-                                                                  {0.73155, 0.40837, 0.37126},
-                                                                  {0.6897,  0.5202,  0.323  },
-                                                                  {0.8262,  0.42976, 0.49478},
-                                                                  {0.88775, 0.26835, 0.56724},
-                                                                  {0.85479, 0.08196, 0.51712},
-                                                                  {0.8965, -0.0288,  0.5674 },
-                                                                  {0.75907, 0.05834, 0.39111},
-                                                                  {0.7356, -0.0699,  0.3552 }};
-
+    voronoi::VoronoiDiagram::PointVector data = {{0.85991, 0.6099,  0.54993},
```

Clang (14.0.6)

[warning] variable 'data' is not initialized

(cppcoreguidelines-init-variables)